internal/db/schema/postgres_migration.gen.go

```diff
@@ -6041,6 +6041,38 @@ create table session_credential_dynamic (
   $$ language plpgsql;
   create trigger revoke_credentials after insert on session_state
     for each row execute procedure revoke_credentials();
+`),
```

Comment:
It looks like you need to run `make migrations` again.

Reply:
good catch